tsconfig.json

```diff
@@ -1,14 +1,12 @@
 {
-    "compilerOptions": {
-        "outDir": "./dist/",
-        "sourceMap": true,
-        "noImplicitAny": true,
-        "module": "commonjs",
-        "target": "esnext",
-        "jsx": "react",
-        "lib": ["dom", "esnext"]
-    },
-    "include": [
-        "./src/**/*"
-    ]
-}
```

Comment:
Had set a rule in here and then moved it to `tslint.json` so the formatting changed otherwise same file.